modin/_compat/pandas_api/py36/series.py

```diff
@@ -0,0 +1,161 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Module for 'Python 3.6 pandas' compatibility layer for Series."""
+
+import numpy as np
+import pandas
+
+from ..abc.series import BaseCompatibleSeries
+from modin.utils import _inherit_docstrings
+
+
+@_inherit_docstrings(pandas.Series)
+class Python36CompatibleSeries(BaseCompatibleSeries):
```

Comment:
It seems to me that the definition of `explode` function for this class is missing, isn't it?

Reply:
It's defined in `base`, though I wonder why it's different - this piece was @devin-petersohn deed, tho :)

Reply:
I see `explode` function in `modin/_compat/pandas_api/latest/base.py` but not in `modin/_compat/pandas_api/py36/base.py`. Is that correct?